pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -821,9 +815,9 @@ func (t *Test) plan() (plan *TestPlan, retErr error) {
 		}
 	}()
 	var retries int
-	// In case the length of the test plan exceeds `opts.maxNumPlanSteps`, retry up to 100 times.
+	// In case the length of the test plan exceeds `opts.maxNumPlanSteps`, retry up to 200 times.
 	// N.B. Statistically, the expected number of retries is miniscule; see #138014 for more info.
-	for ; retries < 100; retries++ {
+	for ; retries < 200; retries++ {
```

Comment:
Had to bump this up for the max_numSteps test to pass.

Reply:
Nevermind, I just made a typo that inadvertently affected the RNG. Given how brittle this test is at 100 retries I still think we should bump it though?

Reply:
Sure. Let's see how long that lasts :) Eventually, we'll probably have to fix the unit test(s); I suspect RNG diverged due to another unit test not fully restoring the initial state.